servicesExtensions/requestSyncrhoneExtensionEscrow_service.py

```diff
@@ -6,13 +6,33 @@
 from servicesCore.requestCore_service import RequestCoreService
 from servicesExternal.web3_single import Web3Single
 
-
+requestCoreArtifact = Artifacts.requestCoreArtifact
+requestSynchroneExtensionEscrowArtifact = Artifacts.requestSynchroneExtensionEscrowArtifact
 class RequestSynchroneExtensionEscrowService:
     def __init__(self):
-        pass
+        global requestCoreArtifact, requestSynchroneExtensionEscrowArtifact
+        self._web3Single = Web3Single.getInstance()
+        self._abiRequestCore = requestCoreArtifact.abi
```

Comment:
The `artifacts` need to be interacted with like `dict`s, so you'll need to do `requestCoreArtifact['abi']`